src/utils/data/getPropertyValues.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * @flow
+ */
+
+import { Map, set } from 'immutable';
+import { Models } from 'lattice';
+
+import getPropertyValue from './getPropertyValue';
+
+const { FQN } = Models;
+
+export default function getPropertyValues(
+  entity :Map | Object,
+  fqns :Array<FQN | string>,
+  fallback ?:boolean | number | string = '',
+) :Object {
+
+  let propertyValues = {};
+
+  fqns.forEach((fqn :FQN | string) => {
+    const value = getPropertyValue(entity, fqn, fallback);
+    propertyValues = set(propertyValues, fqn, value);
+  });
```

Comment:
I think `fallback` here should be an `Array<boolean | number | string>` to account for each fqn in the `fqns` parameter. In the `forEach` block we can then access each fallback value by index.

But since `fallback` is optional, if it is undefined we should initialize it to an empty array of size `fqns`:
```
// disable eslint (reassign parameters )
if (!isDefined(fallback)) {
  fallback = new Array(fqns.length) // empty array of fqns.length undefined's 
}
```
then:

```
fqns.forEach((fqn :FQN | string, index) => {
    const value = getPropertyValue(entity, fqn, fallback[index]);
    propertyValues = set(propertyValues, fqn, value);
});
```

Reply:
I'm not totally sure about this. Having to specify an array of fallbacks is not ideal, especially since all this function does is simply just a `forEach`... why can't the caller just call `getPropertyValue` instead?

Reply:
actually ended up removing `getPropertyValues`! discussed this with hristo in this thread: https://github.com/openlattice/lattice-utils/pull/175#discussion_r514607528